pkg/apiserver/auditing/backend.go

```diff
@@ -0,0 +1,118 @@
+package auditing
+
+import (
+	"bytes"
+	"context"
+	"crypto/tls"
+	"encoding/json"
+	"k8s.io/klog"
+	"net/http"
+	"os"
+	"os/signal"
+	"time"
+)
+
+const (
+	WaitTimeout = time.Second
+	WebhookURL  = "https://kube-auditing-webhook-svc.kubesphere-logging-system.svc:443"
```

Comment:
Why it's hardcoded, I think we have auditoptions https://github.com/kubesphere/kubesphere/blob/master/pkg/simple/client/auditing/elasticsearch/options.go for this